scripts/ai/controllers/BalerController.lua

```diff
@@ -28,6 +28,10 @@ function BalerController:init(vehicle, baler)
     self.baleWrapperSpec = self.baler.spec_baleWrapper
     self.lastDroppedBale = CpTemporaryObject()
     self:debug('Baler controller initialized')
+    local additives = self.balerSpec.additives

```

Comment:
Does it make sense to move additives to the ImplementController so we don't need this boilerplate code everywhere?